shared/actions/config/index.tsx

```diff
@@ -500,6 +502,37 @@ function* criticalOutOfDateCheck() {
   }
 }
 
+// onUpdateUserReacjis hooks `userReacjis`, frequently used reactions
```

Comment:
actually maybe this whole thing should be in chat and not config. We get a static chat config that maybe should hold this data

Reply:
cc: @mmaxim

Reply:
moved `userReacjis` over to the chat store, @mmaxim said the static config isn't the right place so chained to the bootstrap action